mod-source-record-storage-server/src/main/java/org/folio/consumers/QuickMarcKafkaHandler.java

```diff
@@ -67,22 +65,23 @@ public Future<String> handle(KafkaConsumerRecord<String, String> consumerRecord)
     log.trace("handle:: Handling kafka consumerRecord {}", consumerRecord);
 
     var kafkaHeaders = consumerRecord.headers();
-    var params = new OkapiConnectionParams(kafkaHeadersToMap(kafkaHeaders), vertx);
+    var okapiHeaders = toOkapiHeaders(kafkaHeaders, null);
```

Comment:
i'd create overloaded version of toOkapiHeaders() for this case:
```
public static Map<String, String> toOkapiHeaders(List<KafkaHeader> kafkaHeaders) {
    return toOkapiHeaders(kafkaHeaders, null);
}
```

Reply:
agree, done